vignettes/LionessApplicationinGTExData.Rmd

```diff
@@ -21,16 +23,7 @@ LIONESS (Linear Interpolation to Obtain Network Estimates for Single Samples) is
   
 In this vignette, we will compare LIONESS regulatory networks from 207 females and 238 males with colon cancer using RNA-Seq data from TCGA. We will compare the edge weights between females and males using a linear regression model and correcting for the covariates age, race, and disease stage, as available in the limma package. We will also compare the gene's in-degree (defined as the sum of the gene's incoming edge weights from all TFs in the network). Finally, we will perform gene set enrichment analysis to find the pathways enriched for genes differentially targeted by sex in colon cancer.
 
-## Install packages
```

Comment:
@lopesramos Camila why did you remove those lines in the LIONESS tutorial? Is it ok to put them back?